recipes/scikit-garden/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "scikit-garden" %}
+{% set version = "0.1.3" %}
+{% set bundle = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash = "d6fe65205f39633f17c181f552a0fc56e2f05bb2096be63f38c160fe20718d9d" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ bundle }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: {{ build }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - cython
+    - numpy 1.7.*   # [py27]
+    - numpy 1.9.*   # [py35]
+    - numpy 1.11.*  # [py36]
+
+  run:
+    - python
+    - numpy >=1.7   # [py27]
```

Comment:
I'd suggest it is unlikely that this is right. Is there a reason not to use ```numpy x.x```? (https://github.com/conda-forge/staged-recipes/wiki/Frequently-asked-questions#4-what-does-numpy-xx-mean)

cc: @betatim

Reply:
The first first (and second) of the docs were too optimistic that more versions of older numpy would be available soon on Windows. The current "best" way to do it is:


```yaml
build:
    - numpy 1.8.*  # [not (win and (py35 or py36))]
    - numpy 1.9.*  # [win and py35]
    - numpy 1.11.*  # [win and py36]
run:
    - numpy >=1.8  # [not (win and (py35 or py36))]
    - numpy >=1.9  # [win and py35]
    - numpy >=1.11  # [win and py36]
```

Hopefully we'll be able to get a common minimum of `numpy 1.8` (and in for py27 and py34 `numpy 1.7`).
CAVEAT: You still need to respect the minimum `numpy` for the package in question. The fact that the ABI is consistency does not mean you can build a package that requires `numpy 1.10` with `numpy 1.8`.